### PR TITLE
Useful debug help

### PR DESCRIPTION
**Pull request by jtaleric**
**Description of the change:**

Simple PR to help users debugging Ansible errors

**Motivation for the change:**

Help users debug issues

### CODE REVIEW

File: doc/ansible/user-guide.md [hunk 1]
```diff
@@ -395,6 +395,26 @@ kubectl logs deployment/memcached-operator -c operator
...
+          env:
+           ...
+           - name: ANSIBLE_VERBOSITY
+             value: "4"
+```
+
+Additionally, to view formatted output, you can run,
+
+```
+$  oc exec <operator-pod> -- cat /tmp/ansible-operator/runner/<crd group>/v1alpha1/<kind>/<namespace>/<cr>/artifacts/latest/stdout
```
Comment (fabianvf) [inline doc/ansible/user-guide.md]: I would probably remove this, or at least note again that if you want to you can aggregate all those logs with a sidecar we built: https://github.com/operator-framework/operator-sdk/pull/1352/files#diff-2a1f94c1f2f65c1266ac5d5d482009c7R391

**Comment by fabianvf** [comment]
Just one comment but other than that looks great, thanks a ton for the contribution!

File: doc/ansible/user-guide.md [hunk 2]
```diff
@@ -395,6 +395,26 @@ kubectl logs deployment/memcached-operator -c operator
 The `ansible` logs contain all of the information about the Ansible run and will make it much easier to debug issues within your Ansible tasks,
 whereas the `operator` logs will contain much more detailed information about the Ansible Operator's internals and interface with Kubernetes.
 
+### Additional Ansible debug
+
+Occasionally while developing additional debug in the Operator logs is nice to have. To enable Ansible debug output, ie `-vvvv`.
+Add the following to the Operator manifest.
```
Comment (shawn-hurley) [inline doc/ansible/user-guide.md]: can we maybe add `operator.yaml` to this line? It took me a second to figure out what we were referring to.

File: doc/ansible/user-guide.md [hunk 3]
```diff
@@ -288,7 +288,7 @@ $ export OPERATOR_NAMESPACE=$(kubectl config view --minify -o jsonpath='{.contex
 $ sed -i "s|REPLACE_NAMESPACE|$OPERATOR_NAMESPACE|g" deploy/role_binding.yaml
 ```
 
-**Note**  
+**Note**
```
Comment (jmrodri) [inline doc/ansible/user-guide.md]: Thanks for removing the trailing spaces. I hate trailing spaces! :+1:

**Comment by jtaleric** [comment]
> ACK once the suggestions @shawn-hurley and @fabianvf made are addressed, this can be merged.

Good to go now. Sorry for the delay!

**Pull request merged**